src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub/files/jupyterhub/03-profiles.py

```diff
@@ -103,22 +103,30 @@ def base_profile_shared_mounts(groups):
             {"name": "shared", "persistentVolumeClaim": {"claimName": shared_pvc_name}}
         )
 
+    relevant_groups = []
```

Comment:
Minor stylistic nitpick. `relevant_groups` is not very specific which somewhat hurts readability. Something more specific like `groups_to_volume_mount` would be more descriptive and make it easier to follow what is going on.

Reply:
nice catch, I addressed in the new commit